src/Azure.Functions.Cli/Helpers/AzureHelper.cs

```diff
@@ -464,6 +465,11 @@ public static async Task PrintFunctionsInfo(Site functionApp, string accessToken
             functions = await GetFunctions(functionApp, accessToken, managementURL);
 
             ColoredConsole.WriteLine(TitleColor($"Functions in {functionApp.SiteName}:"));
+            if (functionApp.IsKubeApp && !functions.value.Any())
+            {
+                ColoredConsole.WriteLine(WarningColor(
+                    $"The deployment success. Wait for {Constants.DefaultRestartedWorkerProcessUptimeWithin / 1000} sec to fetch the trigger info after the function app provisioning has been finished."));
```

Comment:
change to 
`The deployment succeeded. Waiting for {Constants.DefaultRestartedWorkerProcessUptimeWithin / 1000} sec to fetch the list of functions deployed in the function app.")`

Reply:
Thank you, @pragnagopa 
I love the sentence. Updated.